packages/wdio-appium-service/src/launcher.js

```diff
@@ -48,8 +49,12 @@ export default class AppiumLauncher {
     }
 
     _startAppium(command, args, waitStartTime, callback) {
-        log.debug(`Will spawn Appium process: ${command} ${args.join(' ')}`)
-        let process = spawn(command, args, { stdio: ['ignore', 'pipe', 'pipe'] })
+        const cmd = isWindows ? 'cmd' : command
+        if(isWindows){
+            args.unshift('/c', command)
```

Comment:
I'd prefer to use `path.parse(process.cwd()).root` to identify the operating system's root but it's not mandatory I suppose